nailgun/entity_mixins.py

```diff
@@ -510,6 +511,44 @@ def __repr__(self):
             )
         )
 
+    def to_json(self):
+        r"""Create a JSON encoded string with Entity properties. Ex:
```

Comment:
@abalakh : this is for Sphinx and Flake do not fail when reading my string. I got Travis errors while generating doc and solved after checking it on [SO](http://stackoverflow.com/questions/1137161/what-is-the-correct-way-to-document-a-kwargs-parameter):